torch/_inductor/codegen/rocm/ck_universal_gemm_template.py

```diff
@@ -420,3 +403,23 @@ def add_ck_gemm_choices(
                 choices,
                 op=op,
             )
+
+    def size_args(self):
+        X = self.input_nodes[0]
+        W = self.input_nodes[1]
+        Bias = self.input_nodes[2] if len(self.input_nodes) > 2 else None
+        Y = self.output_node
+
+        M = sympy.expand(X.get_size()[0])
```

Comment:
qq: what does`sympy.expand` do? I'm thinking if there's a better alternative in our big dynamic shape library =p

for example, there's this from the PT2 core library:
https://github.com/pytorch/pytorch/blob/c17d26c3c1b86e9ed7827e1a6e1a3dcc810b5ff8/torch/fx/experimental/symbolic_shapes.py#L4679

there's also this in Inductor which is very similar to above:
https://github.com/pytorch/pytorch/blob/c17d26c3c1b86e9ed7827e1a6e1a3dcc810b5ff8/torch/_inductor/sizevars.py#L91-L92

Reply:
I think I took `expand` from some part of triton kernel code. Not sure which method here is actually correct

Reply:
Okay I think it's okay to use `sympy.expand` then. 

If this is used by codegen then we may benefit from sizevar's `simplify` which will substitute all symbols in `self.replacements` to make `expr` more canonical.

Reply:
Looks like it would make sense to move the simplification to only kernel call site